docs/src/basics/solution.md

```diff
@@ -90,7 +90,7 @@ sol(t)
 Note that the interpolating function allows for `t` to be a vector and uses this to speed up the interpolation calculations. The full API for the interpolations is
 
 ```julia
-sol(t,deriv=Val{0};idxs=nothing,continuity=:left)
```

Comment:
that's not a kwarg: it's before the `;`. That's an optional argument.

Reply:
Ok. However, does the syntax work for you?

Reply:
yes, `sol(t,Val{0})` works